SETUP.md

```diff
@@ -62,53 +62,41 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
   1. (You can reinstall with your updated version after you clone the repository if necessary) Reinstall node_modules `cd apps; yarn; cd ..`
 1. (El Capitan) Ensure that openssl is linked: `brew link --force openssl`
 
-
-### Ubuntu 14.04
+### Ubuntu 17.04 ([iso][ubuntu-iso-url])
```

Comment:
Can you change this to `[download iso]`? I just think it's unexpected that it's a download and not a link/kinda annoying.